docs/CodeOwners.md

```diff
@@ -0,0 +1,112 @@
+# Code Owners
+
+This file outlines the owners of the FHIR Server features in OSS.
+
+### Portal
```

Comment:
Should we have Portal as a section in OSS?

Reply:
Good point - removed in 4eac110.